### DISCUSSION TITLE
how to change party name text color with class - Class Coloured party name

### DISCUSSION THREAD

**Discussion by css773** [kaytotes/ImprovedBlizzardUI#187] [Q&A]
Hi mate ,
I want to achieve an effect - TBC version

e.g. The first player is mage , name color is sky-blue
       The second player is priest , name color is white

i found some codes within \modules\frames\party.lua   , how do i change the codes ?

 for i = 1, 4 do
        -- Update Fonts
        _G["PartyMemberFrame"..i.."Name"]:SetTextColor(font.r, font.g, font.b, font.a);
        _G["PartyMemberFrame"..i.."Name"]:SetFont(font.font, 10, font.flags);

**Comment by kaytotes** [comment]
I do not know the solution to this and is likely more involved than you think.
